Sources/ArrayExtensions.swift

```diff
@@ -201,6 +203,16 @@ extension Array where Element: Equatable {
     }
 }
 
+extension Array where Element: Hashable {
+
+    /// EZSE: Removes all occurrences of the given object(s)
+    public mutating func removeAll(_ elements: [Element]) {
+        let elementsSet = Set(elements)
```

Comment:
This operation is a bag transformation to a set. E.g [1, 1, 2, 3, 2] would become [1, 2, 3]

I am not inclined towards the contract of removeAll to also include a bag -> set transform.